.github/workflows/pr_review.yml

```
@@ -0,0 +1,16 @@
+name: Refine PRs
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  label_issue:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Add default reviewers to PR"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ISSUE_URL: ${{ github.event.pull_request.html_url }}
+        run: |
+          gh pr edit $PR --add-reviewer @atlassianps/reviewers
```

```suggestion
          ISSUE_URL: ${{ github.event.pull_request.html_url }}
        run: |
          gh pr edit $ISSUE_URL --add-reviewer @atlassianps/reviewers
```

I think you want the variable names on lines 14 and 16 to match